Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md

```diff
@@ -123,77 +123,115 @@ The list should be maintained in alphabetic order. DELETE This Row
 
-`C / R`
+`E` = Embodied emissions of a given piece of software
 
-`C` = The total amount of carbon the software is emitting over a time period.
+**These are used to calculate total carbon emissions (`C`):**
 
-`R` = The other measure we will be comparing against.
+`C` = Total amount of carbon the software is emitting over a time period.
 
-## Taxonomy of Green Software
+`C = O + Embodied `
+
+
+## Taxonomy & Terminolopgy of Green Software
 
 Green Software broadly addresses emissions in these categories.
 
-***Operational Emissions:***
+***Operational Emissions (`O`):***
+To calculate the operational emissions associate with software, multiply the electricity consumption of the hardware the software is running on by the regional, granular marginal emissions rate. Because this standard uses a consequential approach, marginal emissions rates should be used for electricity consumption. The marginal emissions rate reflects the change in emissions assoicated with a change in demand. 
+
+Carbon aware software will optimize the timing and location of operation to minimize emissions associated with operation. This could consist of moving computation to regions with cleaner grid emissions or delaying jobs to cleaner periods (or a combination of both). Energy efficient software will also run on hardware that requires less energy to operate or the software can be re-architected to require less energy to execute. All combined, these effects should be reflected in the total operational emissions. 
+
+### Energy Measurement (`E`) 
+This is a reflection of the energy consumption consumed by a given piece of software for a given task. This could be applied for several taxonomies:  
+- Datacenter
+- Indiviudual machine (e.g. VM/Node)
+- Indiviudual service (e.g. machine learning training job)
+- Execution of code 
+
+### Location-Based Carbon Intensity Measurement (`I`)
+The carbon intensity of electricity is a measure of how much carbon (CO2eq) emissions are produced per kilowatt-hour (kWh) of electricity consumed, for a standard unit of gCO2eq/kWh.
+
+Location-based measures the grid carbon intensity (annual average) of a regional balancing authority. From a developer perspective, only the location-based info is important for having an impact on reducing carbon emissions. This excludes market-based measures, and is distinct from 100% renewable energy claims. 
+
+The only figure that matters if you’re trying to optimize the scheduling of your compute in real-time is the marginal emissions intensity. This is the emissions intensity of the marginal power plant which will be turned up if you schedule some compute (e.g. increase electricity demand from the grid) at that moment.
+
+Some categories of applications that apply Operational Emissions (O): 
+
 - **Carbon Aware Application**: Applications that change behaviour to use the cleanest energy possible, for instance a laptop that charged only when there is lots of renewable power currently available.
```

Comment:
I'm now starting to think based on our conversations, the best way to describe this "taxonomy" is not as a category of application, but as the actions, you can take to reduce emissions. Once you merge this I would like to create a PR to move those out into a new section called "actions" where we list the 3 top-level buckets of actions you can take to reduce emissions.

- Use less electricity.
- Use less hardware.
- Use electricity more intelligently.

Reply:
And use electricity more intelligently = using it in different places or at different times